4-Express/server/handlers/storage/setItem.js

```diff
@@ -0,0 +1,20 @@
+const path = require('path')
+const db = require(path.resolve('server/db/','db'))
+
+const setItem = (req,res,next) => {
+    if(!req.body||!req.body.id)
```

Comment:
have you considered using optional chaining? https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining